version-control/git-mac.md

```diff
@@ -1,33 +1,23 @@
-Install Homebrew - a package manager to install and manage development software you'll use.
+## Install Git on Mac
 
-In Google Chrome, navigate to http://brew.sh/
+We will use the Homebrew Package Manager we installed in the Node & npm section to install Git. 
```

Comment:
Should we have the attendees check that they don't have Git first before they install? (or have them update git instead)?